packages/apiv2/auth.go

```diff
@@ -58,18 +59,19 @@ func jwtToken(r *http.Request) (*jwt.Token, error) {
 func jwtGenerateToken(w http.ResponseWriter, claims JWTClaims) (string, error) {
 	token := jwt.NewWithClaims(jwt.SigningMethodHS256, claims)
 	return token.SignedString([]byte(jwtSecret))
-	//	w.Header().Set("Authorization", jwtPrefix+signedToken)
 }
 
-func authWallet(w http.ResponseWriter, r *http.Request, data *apiData) error {
+func authWallet(w http.ResponseWriter, r *http.Request, data *apiData, logger *log.Entry) error {
 	if data.wallet == 0 {
+		logger.Error("wallet is empty")
```

Comment:
Where the log type? May be it should be something like
logger.WithFields(log.Fields{"type": consts.EmptyParam}).Error("wallet is empty")?